SU2_CFD/include/numerics/turbulent/turb_sources.hpp

```diff
@@ -716,6 +712,8 @@ class CSourcePieceWise_TurbSST final : public CNumerics {
     AD::SetPreaccIn(PrimVar_Grad_i, nDim + idx.Velocity(), nDim);
     AD::SetPreaccIn(Vorticity_i, 3);
     AD::SetPreaccIn(V_i[idx.Density()], V_i[idx.LaminarViscosity()], V_i[idx.EddyViscosity()]);
+    AD::SetPreaccIn(Coord_i[1]);
+    AD::SetPreaccIn(V_i[idx.Velocity() + 1]);
```

Comment:
You can remove these 2 lines from ResidualAxisymmetric

Reply:
ok, done!